docs/developer/extending/apps/building-payment-app.mdx

```diff
@@ -0,0 +1,856 @@
+```ts
+// src/pages/api/manifest.ts
+import { createManifestHandler } from "@saleor/app-sdk/handlers/next";
+import { AppManifest } from "@saleor/app-sdk/types";
+import { transactionInitializeSessionWebhook } from "./webhooks/transaction-initialize-session";
+// highlight-next-line
+import { transactionProcessSessionWebhook } from "./webhooks/transaction-process-session";
+
+export default createManifestHandler({
+  async manifestFactory({ appBaseUrl }) {
+    // ...
+    const manifest: AppManifest = {
+      name: "Dummy Payment App",
+      webhooks: [
+        transactionInitializeSessionWebhook.getWebhookManifest(appBaseUrl),
+        // highlight-next-line
+        transactionProcessSessionWebhook.getWebhookManifest(appBaseUrl),
+      ],
+      // ...
+    };
+  },
+});
+```
+
+Once again, **remember to update the app's webhooks or reinstall the app in Saleor**. Otherwise, the new webhook handler won't be called on `transactionProcess`.
+
+:::warning
+
+When installing the app back, you may get "**App with the same identifier is already installed**" error.
+
+Saleor throws this error because the process of fully uninstalling the app is handled by an asynchronous Saleor worker. This means it can take some time before you can install the app with the same identifier again.
+
+If you encounter this error, you can temporarily change the app's `id` in the App Manifest.
+
+:::
+
+### Calling the `transactionProcess` Mutation
+
+To test the `TRANSACTION_PROCESS_SESSION` webhook, we need to call the `transactionProcess` mutation. The mutation requires the `transactionId` variable, which is the ID of the transaction we want to process. That will be the `id` field from the `transaction` object returned by the `transactionInitialize` mutation.
+
+Here is what the mutation looks like:
+
+```graphql
+mutation TransactionProcess($transactionId: ID!) {
+  transactionProcess(id: $transactionId) {
+    transaction {
+      id
+    }
+    transactionEvent {
+      id
+      type
+    }
+    errors {
+      field
+      message
+      code
+    }
+  }
+}
+```
+
+In response, we should receive the processed transaction and the corresponding transaction event with the `CHARGE_SUCCESS` type:
+
+```json
+{
+  "data": {
+    "transactionProcess": {
+      "transaction": {
+        "id": "UHJvamVjdFRy"
+      },
+      "transactionEvent": {
+        "id": "VHJhbnNhY3Rpb25FdmVudDoz",
+        "type": "CHARGE_SUCCESS"
+      },
+      "errors": []
+    }
+  }
+}
+```
+
+And that concludes the implementation of the Dummy Payment Gateway. With just two webhooks, we managed to model a 3D Secure payment flow.
+
+For most checkouts, placing payment will be the last step. That means you could now [complete the checkout](api-reference/checkout/mutations/checkout-complete.mdx) and place the order 🎉.
+
+## Next steps
+
+Congratulations on building your first Saleor Payment App!
+
+Its scope doesn't have to end here. You can further extend the app with additional features:
```

Comment:
Would be also good to include the generic link to the transactionAPi docs: https://saleor-docs-git-payment-app-guide-saleorcommerce.vercel.app/docs/developer/payments
Also the part responsible for re-calculating the amounts for transaction: https://saleor-docs-git-payment-app-guide-saleorcommerce.vercel.app/docs/developer/payments#recalculations-of-transaction-amounts